tests/test_security.py

```
@@ -348,22 +355,9 @@ def test_malicious_storage_paths(self):
             "security": {"trusted_proxies": ["127.0.0.1"], "excluded_paths": []}
         }
         
-        # This should work (might create directories) but should resolve to a safe location
-        try:
+        # Relative traversal outside the working tree should be rejected.
+        with pytest.raises(ValueError):
```

Ruff (0.15.13)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)